package.json

```diff
@@ -31,7 +32,11 @@
     "docs:dist": "npm run clean:dist && NODE_ENV=production webpack && npm run sassdoc",
     "docs:dist:gzip": "npm run docs:dist && gulp",
     "start": "npm run build && npm run storybook",
-    "test": "npm run lint && npm run jest",
+    "start:native": "(cd ./native && npm run storybook)",
```

Comment:
Perhaps just `"native"` instead of `"start:native"`?

Reply:
I'm not sure 🤔  I went with this way to match the convention in the other scripts e.g. `postinstall / postinstall:native`, `test / test:native`, so it would be good to preserve that. It would be cleaner just to do `npm run native` though..

Reply:
The convention is `task:subtask` i.e. by virtue of running `task` all `subtasks` are executed.

`native` isn't a subtask of `start`, they are mutually exclusive.

Reply:
Ahh ok, I didn't realise that was the convention.

Reply:
Done